ui/src/components/Publication/VersionsAccordion/VersionsAccordion.tsx

```diff
@@ -43,127 +41,88 @@ const VersionsAccordion: React.FC<Props> = (props) => {
     );
 
     return (
-        <Framer.AnimatePresence>
-            <Framer.motion.div
-                id={props.id}
-                key={props.id}
-                className="border-grey rounded shadow transition-colors duration-500 dark:bg-grey-900"
-                initial={{ opacity: 0, height: 0 }}
-                animate={{ opacity: 1, height: 'auto' }}
-            >
-                <Components.Button
-                    title="Versions"
-                    className={`w-full justify-between bg-grey-50 px-6 py-1 font-inter transition-all duration-300 children:border-0 dark:bg-grey-700 ${
-                        expanded ? 'rounded-none rounded-t' : ''
-                    }`}
-                    endIcon={
-                        <OutlineIcons.ChevronDownIcon
-                            className={`w-4 transition-transform duration-300 dark:text-white-50 ${
-                                expanded ? 'rotate-180' : 'rotate-0'
-                            }`}
-                        />
-                    }
-                    onClick={() => setExpanded((prevState) => !prevState)}
-                >
-                    Versions
-                </Components.Button>
-
-                <Framer.AnimatePresence>
-                    {expanded && (
-                        <Framer.motion.div
-                            key="versions-accordion-items"
-                            className="overflow-hidden"
-                            initial={{ height: 0 }}
-                            animate={{ height: 'auto' }}
-                            exit={{ height: 0 }}
+        <Components.AccordionSection id={props.id} title={'Versions'}>
+            <div className="space-y-3 px-6 py-4">
+                {canEditDraft &&
+                    (latestVersion?.currentStatus === 'LOCKED' ? (
+                        <Components.Link
+                            title="Edit Draft"
+                            className="flex w-fit rounded border-transparent text-sm font-semibold text-teal-600 outline-0 transition-colors duration-500 hover:underline focus:overflow-hidden focus:ring-2 focus:ring-yellow-400 dark:text-teal-400"
+                            href={`${Config.urls.viewPublication.path}/${props.selectedVersion.versionOf}/edit?step=0`}
+                            onClick={async (e) => {
+                                e.preventDefault();
+                                await props.onUnlockPublication();
```

Comment:
Could drop the async/await and just return props.onUnlockPublication(), which returns a promise anyway

Reply:
Yeah, that's neater. Just pushed.